README.md

```diff
@@ -0,0 +1,355 @@
+absolutely love it.
+Such an easy way to create websites quickly and easily. And you don't even have to write any JavaScript or PHP!
+Let's use is for this tutorial as well. Search for `Blazor Server App` and click `Next`.
+
+![02_create_a_new_project.jpg](https://mongodb-devhub-cms.s3.us-west-1.amazonaws.com/02_create_a_new_project_53da789c4c.jpg "Project template list in Visual Studio")
+
+Choose a `Project name` and `Location` of you liking.
+I like to have the solution and project in the same directory but you don't have to.
+
+![03_configure_your_new_project.jpg](https://mongodb-devhub-cms.s3.us-west-1.amazonaws.com/03_configure_your_new_project_baec3276cd.jpg "Project Configuration screen in Visual Studio")
+
+Choose your currently installed .NET framework (as described in `Pre-requisites`) and leave the rest on default.
+
+Hit `Create` and you are good to go!
+
+![04_additional_information.jpg](https://mongodb-devhub-cms.s3.us-west-1.amazonaws.com/04_additional_information_47949f55f7.jpg "Additional information for the project")
+
+## Adding the MongoDB Driver to the project to connect to the database
+
+Before we start getting into the code, we need to add one NuGet package to the project,
+the [MongoDB Driver](https://www.mongodb.com/docs/drivers/).
+The Driver is a library that let's you easily access
+your [MongoDB Atlas cluster](https://www.mongodb.com/basics/clusters) and work with your database.
+Click on `Project` -> `Manage NuGet Packages...` and search for `MongoDB.Driver`.
+
+![07_manage_nuget_packages.jpg](https://mongodb-devhub-cms.s3.us-west-1.amazonaws.com/07_manage_nuget_packages_664b480003.jpg "File Menu in Visual Studio")
+
+During that process you might have to install additional components like the ones shown in the following screenshot.
+Confirm this installation as we will need some of those as well.
+
+![08_install_additional_components.jpg](https://mongodb-devhub-cms.s3.us-west-1.amazonaws.com/08_install_additional_components_d8a428df2c.jpg "Preview of the packages that are about to be installed.")
+
+Another message you come across might be the following license agreements which you need to accept to be able to work
+with those libraries.
+
+![09_accept_licenses.jpg](https://mongodb-devhub-cms.s3.us-west-1.amazonaws.com/09_accept_licenses_21361afe2d.jpg "Licenses for the libraries that are about to be installed.")
+
+## Creating a new MongoDB Atlas cluster and database to host our Crypto News
+
+Now that we've installed the driver, let's go ahead and create a cluster and database to connect to.
+
+When you register a new account you will be presented with the selection of a cloud database to deploy.
+Open the `Advanced Configuration Options`.
+For this tutorial we only need the forever-free shared tier. Choose Azure in the list of cloud providers and a
+region of your liking.
+Remember that choice so you can later on deploy your website to the same region on Azure.
```

Comment:
Perhaps explain why the MDB cluster should be in the same region (it does not have not)?
"...deploy your website to the same region on Azure to minimize network latency"

Reply:
Yea if you call out Azure, you probably have to explain why. That said, I think you can remove the region specific stuff all together and be fine.

Reply:
I'll add both I guess. :)

Reply:
Actually, I have to mention both, since they both influence latency.
Fixed in 5c2d639f7a53bacba408b20401021e5e747099b5